README.md

```diff
@@ -33,7 +33,7 @@ To run this project locally, complete the following steps:
 
 You can use IBM Cloud Schematics to deploy this Terraform template in the cloud without having a local terraform configuration.   Just use this link:
 
-- [Deploy on IBM Cloud](https://cloud.ibm.com/schematics/workspaces/create?repository=https://github.com/Cloud-Schematics/openshift-dev-cluster&terraform_version=terraform_v0.12)
+[![Deploy to Kubernetes on {{site.data.keyword.cloud_notm}}](../images/Deploy_to_Openshift.svg "Deploy to Kubernetes on {{site.data.keyword.cloud_notm}}")](https://cloud.ibm.com/schematics/workspaces/create?repository=https://github.com/Cloud-Schematics/openshift-dev-cluster&terraform_version=terraform_v0.12){: external}
```

Comment:
2 things I see here:

1. This uses a relative URL for the SVG, this should probably be an absolute url.   In the readme the image link is broken.  If this is being rendered somewhere else, then it's probably broken there too.

    ![image](https://user-images.githubusercontent.com/974294/126170503-061bd35c-ca38-4919-b10f-24ffc366f63c.png)

2. The markdown doesn't look quite right.  At the beginning there looks like an extra `[!`, there's a `{{site.data.keyword.cloud_notm}}` reference, and at the end there's an extra `{: external}` annotation.  Those annotations don't get rendered here on github.  Are they supposed to show up somewhere else?  Those look consistent with IBM Cloud Docs, which has a separate preprocessing process that they user during publication.z

    ![image](https://user-images.githubusercontent.com/974294/126171198-4f5c9fae-3ab9-4f7c-a3bd-8cc6b8d51e64.png)
    ![image](https://user-images.githubusercontent.com/974294/126171458-a99ac272-b231-4b83-b4a1-5bfcf4de3cb4.png)